.github/BOTMETA.yml

```diff
@@ -583,6 +583,7 @@ files:
     authors: resmo
   $modules/cloud/misc/helm.py:
     authors: flaper87
+    deprecated: true
```

Comment:
Hi, there isn't a deprecated field in botmeta.yml, so this line needs removing.

I wonder if we have anything confusion in the documentation, was there something that made you think you needed to update botmeta.yml?

Reply:
Ok, i will removed it

No, the documentation is good and the example too.
I added it like the others changes that felixfontein ask me to removed because ci pipeline not passed. I didn't know that the CI not working so I look into the code others deprecated modules to find what I haved missed.